datastore/lib/bdkd/datastore/util/ds_util.py

```diff
@@ -81,6 +81,17 @@ def _delete_resource_parser():
                         help='Force deleting a published resource')
     return parser
 
+def _get_file_list_parser(enforce=True):
+    """
+    Parser for get-file-list options.
+    """
+    parser = argparse.ArgumentParser(add_help=False)
+    optional_fields = parser.add_argument_group('Optional arguments',
```

Comment:
Any reason to use add_argument_group? Just calling add_argument to parser should be enough for a single argument.